test/utils/datetime.test.js

```diff
@@ -210,4 +209,39 @@ describe('datetime', () => {
         .to.equal('14🙃20');
     });
   });
+
+  describe('formatDisplayDate', () => {
+    const tzAwareLA = {
+      timezoneAware: true,
+      timezoneName: 'America/Los_Angeles',
+    };
+    const tzAwareNY = {
+      timezoneAware: true,
+      timezoneName: 'America/New_York',
+    };
+    const tzUnaware = {
+      timezoneAware: false,
+      timezoneName: null,
+    };
+    const hammertime = 1473048000000; // 09/05/2016 @ 4:00am (UTC)
```

Comment:
Would you mind reformatting the comment in ISO 8601 to avoid confusing people like @jh-bate who aren't used to seeing month first?